hypha/apply/users/templates/two_factor/core/backup_tokens.html

```diff
@@ -30,3 +37,39 @@ <h1>{% block title %}{% trans "Backup Codes" %}{% endblock %}</h1>
   {% endif %}
 
 {% endblock %}
+{% block extra_css %}
+<style>
+@media print {
+    .header, .admin-bar, .actions, .hide-print {
+        display: none;
+    }
+}
+</style>
+{% endblock %}
+
+{% block extra_js %}
+<script src="{% static 'js/clipboard.js' %}"></script>
+<script>
+    {# Select the codes when clicked #}
+    $('#list-backup-tokens').on("click", function (e){
+        $(this).select();
+    });
+
+    {# Setup print button #}
+    $(".btn-print").on("click", function(e){
+        e.preventDefault();
+        window.print();
+    });
+
+    {# Instantiate clipboard by passing a HTML element, uses clipboard.js #}
+    var clipboardBtn = document.querySelectorAll('.btn-copy-to-clipboard');
+    var clipboard = new ClipboardJS(clipboardBtn);
+    clipboard.on('success', function (e) {
+        // @TODO: use a tooptip instead of alert
+        alert("Back codes copied to clipboard.");
```

Comment:
"Backup" and not "Back".

If there is a need to show some confirmation I think we could just flash it quickly. Then the user is not forced to click away something.